src/mscorlib/shared/System/Reflection/ReflectionTypeLoadException.cs

```diff
@@ -24,9 +26,17 @@ public ReflectionTypeLoadException(Type[] classes, Exception[] exceptions, strin
             HResult = HResults.COR_E_REFLECTIONTYPELOAD;
         }
 
+        private ReflectionTypeLoadException(SerializationInfo info, StreamingContext context) 
+            : base(info, context)
+        {
+            LoaderExceptions = (Exception[])(info.GetValue("Exceptions", typeof(Exception[])));
```

Comment:
Why is this not setting Types? MIssing `_classes = (Type[])(info.GetValue("Types", typeof(Type[])));`

Reply:
Types aren't serializable on core

Reply:
Type isn't serializable therefore it would fail...